app/src/main/java/org/wikipedia/gallery/MediaListItem.kt

```diff
@@ -34,11 +34,10 @@ class MediaListItem constructor(val title: String = "",
 
     @Serializable
     @Parcelize
-    class ImageSrcSet : Parcelable {
-
-        @SerialName("scale")
-        private val _scale: String? = null
-        val src: String = ""
```

Comment:
These parameters were not being parceled correctly, and need to be in the primary constructor.